cpu/kinetis_common/doc.txt

```diff
@@ -14,7 +14,7 @@
  *                  #define ADC_NUMOF           (1U)
  *                  #define ADC_0_EN            1
  *                  #define ADC_MAX_CHANNELS    1
- *                  
+ *
  *                  // ADC 0 configuration
```

Comment:
Dammit yes :-) ! Sorry for the redundant talk. If I could chose I would prefer ` / (space) *`. COntentional I can't comment the doc, of course...